nilearn/_utils/testing.py

```diff
@@ -178,7 +179,10 @@ def write_tmp_imgs(*imgs, **kwargs):
         finally:
             # Ensure all created files are removed
             for filename in filenames:
-                os.remove(filename)
+                try:
+                    os.remove(filename)
+                except OSError:
```

Comment:
Why do we need that ?

Reply:
so that the other files are still removed if removing this one fails (eg `os.remove` raises an exception if the file doesn't exist, and the test using this function may have removed the file; it also fails if there is an open file handle for that file on windows)